client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-suggestion.jsx

```diff
@@ -0,0 +1,172 @@
+		};
+		this.state = initialState;
+	}
+	componentWillReceiveProps() {
+		const state = {
+			childSelected: 'default',
+		};
+		this.state = state;
+	}
+
+	_logReasonFollowUp = ( option ) => {
+		this.setState( {
+			childSelected: option.value,
+		} );
+	}
+	setOptions = ( optionValue ) => {
+		let options = [];
+		switch ( optionValue ) {
+			case 'missingFeature':
+				options = [
+					{ value: 'default', label: 'Backup' },
+					{ value: 'missingFeature_followup_themes', label: 'Themes' },
+					{ value: 'missingFeature_followup_other', label: 'Love :)' },
+				];
+				break;
+
+			case 'reason2':
+				options = [
+				{ value: 'default', label: 'sth1' },
+				{ value: 'reason2_followup_sth2', label: 'sth2' },
+				{ value: 'reason2_followup_sth3', label: 'sth3' },
+				];
+				break;
+
+			case 'reason3':
+				options = [
+				{ value: 'default', label: 'sth banana' },
+				{ value: 'reason3_followup_sth2', label: 'sth2' },
+				];
+				break;
+
+			case 'reason4':
+				options = [
+				{ value: 'default', label: 'sth1 reason4' },
+				{ value: 'reason4_followup_sth2', label: 'sth2' },
+				{ value: 'reason4_followup_sth3', label: 'sth3' },
+				];
+				break;
+
+			case 'reason5':
+				options = [
+				{ value: 'default', label: 'sth reason5' },
+				{ value: 'reason5_followup_sth2', label: 'sth2' },
+				];
+				break;
+
+			case 'reason6':
+				options = [
+				{ value: 'default', label: 'sth1 reason6' },
+				{ value: 'reason6_followup_sth2', label: 'sth2' },
+				{ value: 'reason6_followup_sth3', label: 'sth3' },
+				];
+				break;
+
+			case 'reason7':
+				options = [
+				{ value: 'default', label: 'sth reason7' },
+				{ value: 'reason7_followup_sth2', label: 'sth2' },
+				];
+				break;
+
+			case 'reason8':
+				options = [
+				{ value: 'default', label: 'sth reason8' },
+				{ value: 'reason8_followup_sth2', label: 'sth2' },
+				];
+				break;
+		}
+		return options;
+	}
+	setQuestion = ( optionValue ) => {
+		const {
+			translate
+		} = this.props;
+
+		let textShareWhat =	'';
+		switch ( optionValue ) {
+			case 'missingFeature':
+				textShareWhat = translate( 'What feature are you looking for?' );
+				break;
+			case 'reason2':
+				textShareWhat = translate( 'Mind telling us where?' );
+				break;
+			case 'reason3':
+				textShareWhat = translate( 'Mind telling us which one(s)?' );
+				break;
+			case 'reason4':
+				textShareWhat = translate( 'What will you do instead?' );
+				break;
+			case 'reason5':
+				textShareWhat = translate( 'Mind telling us which one(s)?' );
+				break;
+			case 'reason6':
+				textShareWhat = translate( 'Mind sharing more?' );
+				break;
+			case 'reason7':
+				textShareWhat = translate( 'Mind telling us which one(s)?' );
+				break;
+			case 'reason8':
+				textShareWhat = translate( 'What will you do instead?' );
+				break;
+		}
+		return textShareWhat;
+	}
+	render() {
+		const {
+			compactButtons,
+			optionSelected
+		} = this.props;
+		const {
+			childSelected
```

Comment:
Nit: For one or two destructured variables you usually don't need to put them on a separate line, this one would could also do it and will probably be more readable:

```
const { childSelected } = this.state;
```

I've seen this one throughout this PR, feel free to address any other occurrences as well.